aiofreepybox/api/player.py

```diff
@@ -0,0 +1,141 @@
+class Player:
+
+    def __init__(self, access):
+        self._access = access
+        self._api_version_target = 'v6'
+
+    player_volume_data_schema = {
+        'mute': bool(),
+        'volume': int
+    }
+
+    player_url_data_schema = {
+        'url': ''
+    }
+
+    media_control_seek_args = {
+        'seek_position': int,
+        'type': 'seek_position'
+    }
+
+    media_control_stream = {
+        'quality': '',
+        'source': ''
+    }
+
+    media_control_stream_args = {
+        'stream': media_control_stream,
+        'type': 'stream'
+    }
+
+    media_control_track_args = {
+        'track_id': int,
+        'type': 'track_id'
+    }
+
+    media_control_command = [
+        'play',
+        'pause',
+        'play_pause',
+        'stop',
+        'next',
+        'prev',
+        'seek_forward',
+        'seek_backward',
+        'seek_to',
+        'repeat_all',
+        'repeat_one',
+        'repeat_off',
+        'repeat_toggle',
+        'shuffle_on',
+        'shuffle_off',
+        'shuffle_toggle',
+        'record',
+        'record_stop',
+        'select_audio_track',
+        'select_srt_track',
+        'select_stream'
+    ]
+
+    media_control_data_schema = {
+        'args': media_control_stream_args,
+        'cmd': media_control_command[0]
+    }
+
+    async def get_player_status(self, player_id, api_version=None):
+        """
+        Get player status
+
+        player_id : `int`
+        api_version : `str`, optional
+            , by default `self.api_version_target`
+        """
+        if api_version is None:
+            api_version = self.api_version_target
+        return await self._access.get(f'player/{player_id}/api/{api_version}/status/')
```

Comment:
This will generate a url with two `api` fields, like `https://foezimj:28799/api/v6/player/1/api/v6/status/`, is that expected? There is an API version for the Player that is different from the API version of the server?

Reply:
Oh sorry, I just reread your comment above where you explained precisely that

Reply:
yes